mobx/lib/home_page.dart

```diff
@@ -7,102 +7,124 @@ import 'package:mobx_sample/model/todo_list.dart';
 import 'package:mobx_sample/todo_details_page.dart';
 import 'package:provider/provider.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
```

Comment:
Let's keep this a StatelessWidget, this tab does not have to know what tab it belongs to. That should be controlled at a higher level

Reply:
So you are suggesting that I should keep the activeTab in the mobx store maybe a layout store?

Reply:
Ya you can keep an observable in the store like `currentTab` or `activeTab` and switch whenever that changes with an `Observer`

Reply:
Okay I will make the changes